### PR TITLE
add iFlight BLITZ F745/Mini

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-blitz-f745.rst [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    :width: 50%
+
+MINI STACK
+
+*above image and some content courtesy of* `iFlight <http://https://www.iflight.com>`__
+
+Where to Buy
+============
+
+- Available from many retailers including `iFlight <https://shopr.iflight.com>`__
```
Comment (andyp1per) [inline common/source/docs/common-blitz-f745.rst:28]: typo shopr

File: common/source/docs/common-blitz-f745.rst [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+
+Specifications
+==============
+
+-  **Processor**
+
+   -  STM32F745 32-bit processor, 216 MHz
+   -  AT7456E OSD
+   -  32 MB flash for logging
```
Comment (andyp1per) [inline common/source/docs/common-blitz-f745.rst:38]: Mini has 16MB - regular has 32MB

File: common/source/docs/common-blitz-f745.rst [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+OSD Support
+===========
+
+The autopilot  supports OSD using :ref:`OSD_TYPE<OSD_TYPE>` 1 (MAX7456 driver). The defaults are also setup to allow DJI Goggle OSD support on UART4. Both the internal analog OSD and the DisplayPort OSD can be used simultaneously by setting :ref:`OSD_TYPE2<OSD_TYPE2>` = 5 (setup by default)
+
+PWM Output
+==========
+
+The FoxeerF405v2 supports up to 8 PWM outputs. Outputs are available via two JST-SH connectors. All outputs support DShot, as well as all PWM types. The first 4 outputs support bi-directional DShot also.
```
Comment (andyp1per) [inline common/source/docs/common-blitz-f745.rst:113]: Blitz

Comment (andyp1per) [reply common/source/docs/common-blitz-f745.rst:113]: Don't think there are two JST connectors

File: common/source/docs/common-blitz-f745.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-blitz-f745.rst]: fixed

File: common/source/docs/common-blitz-f745.rst (hunk 3 above)
Comment (Hwurzburg) [reply common/source/docs/common-blitz-f745.rst]: fixed

File: common/source/docs/common-blitz-f745.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-blitz-f745.rst]: fixed

**Pull request merged**